src/SIL.XForge.Scripture/ClientApp/src/app/home/home.component.spec.ts

```diff
@@ -29,6 +30,7 @@ describe('HomeComponent', () => {
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
+      imports: [RouterTestingModule],
```

Comment:
I don't think you need this since you are using `[routerLink]`. If you don't need it, revert all changes to this file.

Reply:
if we use this line (imports: [RouterTestingModule],) in home.component.spec.ts file. home component all test cases will be passed otherwise some test cases got failed. that's why we added.